operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfigurations.java

```diff
@@ -0,0 +1,93 @@
+package org.bf2.operator.operands;
+
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.quarkus.runtime.Startup;
+import org.bf2.operator.resources.v1alpha1.ManagedKafka;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.function.UnaryOperator;
+
+/**
+ * Loads the properties for each instance profile from config files and from the Quarkus Config.
+ * <p>
+ * Default properties will have the prefix managedkafka.
+ * <p>
+ * Properties in each instance file will omit this prefix.
+ * <p>
+ * Properties overlayed from the Quarkus Config will be prefixed accordingly - managedkafka.kafka... standard.kafka...
+ */
+@Startup
+@ApplicationScoped
+public class KafkaInstanceConfigurations {
+
+    private static final String MANAGEDKAFKA = "managedkafka";
+
+    public enum Config {
+        STANDARD,
+        DEVELOPER
+    }
```

Comment:
Should this be something like `InstanceType` rather than `Config`? It's cosmetic, but at least for me it looked like a `org.eclipse.microprofile.config.Config` instance was being passed around in this class.